tools/kagami/src/main.rs

```diff
@@ -175,16 +175,39 @@ mod genesis {
     use super::*;
 
     #[derive(StructOpt, Debug, Clone, Copy)]
-    pub struct Args;
+    pub struct Args {
+        /// If set generate synthetic genesis with specified number of domains, accounts and assets.
+        #[clap(short, long, action)]
```

Comment:
I'd imagine that it's much better to phrase this as an `enum`. So `genesis --default` is the default value, `genesis` becomes `genesis --default` and `genesis --synthetic` accepts the three args. 

Just a suggestion. Fix at will.